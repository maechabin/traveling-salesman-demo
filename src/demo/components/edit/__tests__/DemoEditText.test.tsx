import React from 'react';
import { shallow } from 'enzyme';

import DemoEditText from '../DemoEditText';

describe('<DemoEditText />', () => {
  it('子コンポーネントをレンダリングすること', () => {
    // setup
    const wrapper = shallow(<DemoEditText />);

    // snapshot
    expect(wrapper).toMatchSnapshot();
  });
});
