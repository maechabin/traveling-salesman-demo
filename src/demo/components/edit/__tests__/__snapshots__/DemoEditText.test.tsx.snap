// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DemoEditText /> 子コンポーネントをレンダリングすること 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DemoEditText />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          出発地、到着地は入力必須です。
        </p>,
        <p>
          また、ルートは2箇所以上の入力が必要です。
        </p>,
      ],
      "style": Object {
        "textAlign": "center",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "出発地、到着地は入力必須です。",
        },
        "ref": null,
        "rendered": "出発地、到着地は入力必須です。",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "また、ルートは2箇所以上の入力が必要です。",
        },
        "ref": null,
        "rendered": "また、ルートは2箇所以上の入力が必要です。",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            出発地、到着地は入力必須です。
          </p>,
          <p>
            また、ルートは2箇所以上の入力が必要です。
          </p>,
        ],
        "style": Object {
          "textAlign": "center",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "出発地、到着地は入力必須です。",
          },
          "ref": null,
          "rendered": "出発地、到着地は入力必須です。",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "また、ルートは2箇所以上の入力が必要です。",
          },
          "ref": null,
          "rendered": "また、ルートは2箇所以上の入力が必要です。",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
